heft-plugins/heft-jest-plugin/src/HeftJestReporter.ts

```diff
@@ -47,13 +48,20 @@ export default class HeftJestReporter implements Reporter {
       Colors.whiteBackground(Colors.black('START')),
       ` ${this._getTestPath(test.path)}`
     );
+
+    // Test duration is not always available, so we'll use a map to track the start time
+    // of each test and diff against the end time
+    this._testStartTime.set(test.path, Date.now());
```

Comment:
`Date.now()` is not meant to be used for performance measurements.  We should be using `performance.now()` or something like 

https://github.com/microsoft/rushstack/blob/276d593379034746da729eb578e117d1497acb49/apps/rush-lib/src/utilities/Utilities.ts#L141-L147

(Maybe this should be exposed via `node-core-library`?)

Reply:
Looks like `performance.now()` is now supported as of Node.js 8.5.0

Reply:
As you said offline, seems that `performance.now()` is available and was since Node 8. Will use that.